scripts/validate/govcms-validate-modules

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2162,SC2046,SC2002,SC2034
+set -euo pipefail
+
+#
+# GovCMS disallowed modules check.
+#
+# This will lint exported configuration files to identify if
+# any disallowed module has been enabled via exports.
+#
+
+GOVCMS_CONFIG_FOLDER_PATH=${GOVCMS_CONFIG_FOLDER_PATH:-"config"}
+GOVCMS_CORE_EXTENTION_FILE_NAME=${GOVCMS_CORE_EXTENTION_FILE_NAME:-"core.extension.yml"}
+GOVCMS_DISALLOWED_MODULES=${GOVCMS_DISALLOWED_MODULES:-}
```

Comment:
Do we have a default list of modules that we want to ensure are disabled?

Reply:
dblog and update are the 2 that come to mind @steveworley

Reply:
Yeah @klonos mentioned, so far dblog and update are the main disallowed modules. I have made them the default value.